client/python/samples/README.md

```diff
@@ -222,12 +237,215 @@ optional arguments:
   --model_name MODEL_NAME Model name to query. default: vehicle-detection
   --model_version MODEL_VERSION Model version to query. default: latest available
   --output_save_path OUTPUT_SAVE_PATH Path to store output.
```

Comment:
We should write what happens if it's not provided. We're printing the results? Try to store it in some default location?